episodes/04-root.md

```diff
@@ -27,23 +27,8 @@ to describe the [RO-Crate Root](https://www.researchobject.org/ro-crate/1.1/root
 }
 ```
 
-:::::::::::::::::::::::::::::::::::::::: callout
```

Comment:
I missed this first time through, but I think we should move this section to where we add the metadata descriptor.